src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -61,6 +71,8 @@ public RazorSourceGenerationContext(GeneratorExecutionContext context)
 
             globalOptions.TryGetValue("build_property._RazorSourceGeneratorDebug", out var waitForDebugger);
 
+            globalOptions.TryGetValue("build_property.SuppressRazorSourceGenerator", out var suppressRazorSourceGenerator);
```

Comment:
You don't strictly need the build_property prefix here -- that's the prefix used for MSBuild properties that are mirrored in. So if you're really worried about something trying to do that and breaking something, that might be bad. (It also might be helpful maybe if you're trying to work around something or for testing.)